src/dealDashboard/dealDashboard.ts

```diff
@@ -51,6 +56,10 @@ export class DealDashboard {
         this.eventAggregator.publish("showMessage", "Deal privacy has been successfully submitted");
       }),
     );
+    //wait until the dao transactions from the contract are there
+    await Utils.waitUntilTrue(() => this.deal.daoTokenTransactions !== undefined);
```

Comment:
both of these need to be located in the components that require this data

Reply:
Done